server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -47,21 +48,32 @@ type workerActivities interface {
 	TerraformInit(context.Context, activities.TerraformInitRequest) error
 	TerraformPlan(context.Context, activities.TerraformPlanRequest) error
 	TerraformApply(context.Context, activities.TerraformApplyRequest) error
-	ExecuteCommand(context.Context, activities.ExecuteCommandRequest) error
+	ExecuteCommand(context.Context, activities.ExecuteCommandRequest) activities.ExecuteCommandResponse
 	Notify(context.Context, activities.NotifyRequest) error
 	Cleanup(context.Context, activities.CleanupRequest) error
 }
 
 type Runner struct {
 	Activities workerActivities
+	JobRunner  runners.JobRunner
 	Request    Request
 }
 
 func newRunner(ctx workflow.Context, request Request) *Runner {
 	var a *activities.Terraform
+
+	runStepRunner := runners.RunStepRunner{
+		Activity: a,
+	}
 	return &Runner{
```

Comment:
Can you initialize this as a pointer instead of passing this to other functions as a pointer?